voyager-publish/test/unit.test.js

```diff
@@ -114,7 +114,7 @@ describe('publisher (unit tests)', () => {
         ieContract,
         recordTelemetry,
         maxMeasurements: 2,
-        pid: 1000 + i,
+        lock: String(1000 + i),
```

Comment:
Let's use a UUID string here too, to test that the SQL column can store strings that long.